core/base/inc/Rtypes.h

```diff
@@ -368,32 +368,13 @@ public:
 #define R__UseDummy(name) \
    class _NAME2_(name,_c) { public: _NAME2_(name,_c)() { if (name) { } } }
 
-#define ClassImpUnique(name,key)                                                                        \
-   namespace ROOT {                                                                                     \
-      /** \cond HIDDEN_SYMBOLS */ TGenericClassInfo *GenerateInitInstance(const name*); /** \endcond */ \
-      namespace {                                                                                       \
-         static int _R__UNIQUE_(_NAME2_(R__dummyint,key)) __attribute__((unused)) =                     \
-            GenerateInitInstance((name*)0x0)->SetImplFile(__FILE__, __LINE__);                          \
-         R__UseDummy(_R__UNIQUE_(_NAME2_(R__dummyint,key)));                                            \
-      }                                                                                                 \
-   }
+#define ClassImpUnique(name, key)                                                                    \
```

Comment:
I thought we wanted to also raise a warning on usage, as we usually do with other deprecations. But, reading your comment in the release notes about deprecation in 6.38 makes me think we will only start raising a warning in 6.38?

Reply:
This is a good thought. This macro is around since many years. Would it be perhaps better to tackle its removal in 3 steps: no-op, then warning (6.38), and then removal (6.40)?